alert-policies/blazor-web-assembly/ApplicationErrorRate.yml

```diff
@@ -0,0 +1,43 @@
+# Name of the alert
+name: Application Error Rate
+
+# Description and details
+description: |+
+  This alert is triggered when the application error rate are higher than 5 for 5 minutes.
```

Comment:
"This alert is triggered when the application error rates are higher than 5 for 5 minutes."